napari/utils/settings/_defaults.py

```diff
@@ -72,13 +89,17 @@ class Config:
     class NapariConfig:
         # Napari specific configuration
         preferences_exclude = [
-            "schema_version" "preferences_size",
+            "schema_version",
```

Comment:
since we're excluding everything but a single field, should we consider having a `preferences_include` instead?

Reply:
@tlambert03 I can make that change if you think its better, but my assumption is in the long run, there will be more settings to include in the user exposed preferences than the ones we don't want to include.  What do you think?

Reply:
> @tlambert03 I can make that change if you think its better, but my assumption is in the long run, there will be more settings to include in the user exposed preferences than the ones we don't want to include. What do you think?

I was kinda assuming otherwise 😄 ... but I don't have strong feeling either way.  @sofroniewn?

Reply:
@tlambert03 @sofroniewn You guys would know way better!  Should I make the change?

Reply:
easy to change later... don't worry about it.  @sofroniewn can weigh in before merge

Reply:
Yeah, we can hold off until we have more examples - i'm actually really not sure which way it'll go, and don't have a strong preference!